core/server/master/src/main/java/alluxio/master/meta/DefaultMetaMaster.java

```diff
@@ -606,6 +606,16 @@ public void resetState() {
     mPathProperties.resetState();
   }
 
+  @Override
+  public void updateConfig(Map<String, String> propertiesMap) {
```

Comment:
as mentioned below, it might be helpful to return the set of successfully updated keys?

Reply:
OK